src/solids4FoamModels/numerics/sparseMatrixExtended/sparseMatrixExtendedTools.C

```diff
@@ -1068,7 +1068,7 @@ void Foam::sparseMatrixExtendedTools::enforceFixedDof
                 coeff(2,1) = momDispCoeff.zy();
                 coeff(2,2) = momDispCoeff.zz();
             }
-
+            
```

Comment:
Be careful of adding whitespace to files. Most code text editors have automated ways to remove or avoid adding this.
Also, before making a commit, it is good practice to review the changes with `git diff` (or the GUI equivalent); if you see whitespace edits you should remove them and not commit them.